libs/core/src/net/server_message.rs

```diff
@@ -17,7 +17,7 @@ pub enum ServerMessagePayload {
     UpdateRoomPlayers(Vec<MultiplayerRoomPlayer>),
     /// Must have the same length as a last sent UpdateRoomPlayers,
     /// contains server (entity) ids for corresponding players.
```

Comment:
This comment should be updated (the vector doesn't have the length constraint anymore)

Reply:
Fixed